src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -34,6 +35,16 @@ var (
 
 const minPollingIntervalSeconds = 60 * 5 // 5 min (News feeds can be genuinely spammy)
 
+// includeRules contains the rules for including or excluding a feed item.
+type includeRules struct {
+	// Author is a list of strings that the author name must match or not match.
```

Comment:
Please state whether there is case sensitivity.

Reply:
Yes, it's case sensitive atm. Added to the docs - let me know if you think case-insensitive is better.

Reply:
Let's see how many people complain with it being case-sensitive :)